docs/configuration-reference/components/headlamp.md

```diff
@@ -0,0 +1,69 @@
+# Headlamp configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Attribute reference](#attribute-reference)
+* [Applying](#applying)
+* [Deleting](#deleting)
+
+## Introduction
+
+[Headlamp](https://github.com/kinvolk/headlamp) is an easy-to-use and versatile
+dashboard for Kubernetes.
+
+It has a clean and modern UI, it is vendor independent, generic, and supports
+the most common operations for Kubernetes clusters.
+
+## Prerequisites
+
+* A Lokomotive cluster accessible via `kubectl`.
```

Comment:
Actually, if it's vendor independent, maybe we should say `Kubernetes cluster` here?